pom.xml

```diff
@@ -87,12 +91,23 @@
             <artifactId>amazon-sqs-java-messaging-lib</artifactId>
             <version>1.0.8</version>
         </dependency>
+        <!-- used to send SNS messages -->
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk-sns</artifactId>
+            <version>1.11.106</version>
```

Comment:
[older version](https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-sns/1.11.106) used bcoz of compatibility issues?

Reply:
yep, because of 
```xml
        <dependency>
            <groupId>com.amazonaws</groupId>
            <artifactId>amazon-sqs-java-messaging-lib</artifactId>
            <version>1.0.8</version>
        </dependency>
```